### ISSUE TITLE
New incompatability with Pimax

### ISSUE THREAD

**Issue by ReadieFur** [mm0zct/Oculus_Touch_Steam_Link#36]
I haven't used this tool in quite some time and I am now coming back to using it, however when I launch the ovr_test program alongside Pimax, SteamVR will crash. This never used to happen on older versions.

**Comment by mm0zct** [comment]
can you try launcing it _after_ steamvr and pimax are running? I had to add this feature for steam vr streaming to work with the quest.

**Comment by ReadieFur** [comment]
> can you try launcing it _after_ steamvr and pimax are running? I had to add this feature for steam vr streaming to work with the quest.

Only just coming back to this now, I can launch ovr_test after steamvr has started but the oculus devices will not appear in SteamVR. I rembember when I went to do the rewrite on this program with you ages ago that it was found that ovr_test has to be launched before SteamVR otherwise devices wont appear, as is the case here.

TL;DR: I can launch ovr_test after SteamVR and Pimax starts but the oculus devices will not appear.

**Comment by ReadieFur** [comment]
Closing this issue as the latest version with external tracking enabled seems to resolve the issue.
